### PR TITLE
feat(apm): Add more manual react profile points

### PR DESCRIPTION
**Pull request by billyvg**
This adds more manual react profile points to

- User feedback
- Releases
- Release details
- Issues list
- Issue details

### CODE REVIEW

**Review by davidenwang — APPROVED** [review]
nice! excited about this

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/sentry/static/sentry/app/views/issueList/overview.jsx [hunk 1]
```diff
@@ -168,6 +172,14 @@ const IssueList = createReactClass({
       // This can happen when transitionTo is called
       this.fetchData();
     }
+
+    if (
+      prevState.issuesLoading &&
+      !this.state.issuesLoading &&
+      typeof this.props.finishProfile === 'function'
+    ) {
+      this.props.finishProfile();
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/issueList/overview.jsx]: You could use `callIfFunction()` to skip the type check :bike:

Comment (billyvg) [reply src/sentry/static/sentry/app/views/issueList/overview.jsx]: I can follow up with this

**Pull request merged**